README.md

```diff
@@ -15,8 +15,11 @@ Make sure you have [node.js](https://nodejs.org/en/) and [npm](https://www.npmjs
 
 Get [zeromq](http://zeromq.org/) using the package manager of your distro. For example here is the package in [Archlinux](https://www.archlinux.org/packages/community/x86_64/zeromq/) and in [Ubuntu](https://packages.ubuntu.com/source/trusty/libs/zeromq).
 
-Also get [sqlcipher](https://www.zetetic.net/sqlcipher/) using the package manager of your distro. For example here is the package in [Archlinux](https://www.archlinux.org/packages/community/x86_64/sqlcipher/) and in [Ubuntu](https://packages.ubuntu.com/trusty/database/sqlcipher).
+Also get [sqlcipher](https://www.zetetic.net/sqlcipher/):
 
+- If you're running Archlinux you can install the [package](https://www.archlinux.org/packages/community/x86_64/sqlcipher/) with the package manager of your distro.
```

Comment:
Thanks! Can you change the wording: `package manager of your distro` in both cases?

As you now have 2 different paragraphs for each distro you can name the package managers.

It's `pacman` for Archlinux and `apt-get` for Ubuntu.

Reply:
I usally prefer the newer apt, but apt-get works fine as well.